src/identify/index.js

```diff
@@ -241,6 +206,9 @@ class IdentifyService {
       protocols: Array.from(this._protocols.keys())
     })
 
+    // TODO: should we add to peerStore.addressBook.set() here?
+    // We can have an inbound connection from an unkwown peer
```

Comment:
No, this function is us responding to a request, identify will run for every connection regardless of direction so we'll get their addresses when it is called.